### PR TITLE
Upgrade / rewrite

### PR DESCRIPTION
**Pull request by aintgoin2goa**
- Brought app into 2016 with nbt config-vars etc
- Copied code from spoor enrichment to split metrics by region - also now uses next-metrics
- Split each service onto a seperate dyno
- Added metrics for falcon service

### CODE REVIEW

**Comment by matthew-andrews** [comment]
nice, +1

File: Makefile [hunk 1]
```diff
@@ -8,12 +9,18 @@ install:
...
-run:
-	export DEBUG=*; \
-	export fastly_apikey=`cat .fastly`; \
-	export fastly_service_prod=`cat .fastly-prod`; \
-	export fastly_service_stage=`cat .fastly-stage`; \
- 	node app;
+test: verify
+
+falcon-stats:
+	forever stats.js --service 4yA0Sk3aWMJwj4WZJlBct1
```
Comment (commuterjoy) [inline Makefile]: i guess we assume service ids are public?

File: stats.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+					'status_301': stats.status_301,
+					'status_302': stats.status_302,
+					'status_304': stats.status_304,
+					'status_503': stats.status_503,
+					'bandwidth': stats.bandwith,
+					'errors': stats.errors
+				};
+
+				Object.keys(m).forEach(k => {
+					logger.verbose(`fastly.${serviceId}.${region}.${k}`, m[k]);
```
Comment (commuterjoy) [inline stats.js]: might break any alerting set up on these keys?

**Comment by commuterjoy** [comment]
+1 aside from comments

File: Makefile (hunk 1 above)
Comment (aintgoin2goa) [inline Makefile]: yeah, I don't think there is anything you could do with them unless you have a login / api key?

**Pull request merged**